pallets/loans/src/tests.rs

```diff
@@ -274,7 +271,7 @@ fn redeem_fails() {
 }
 
 #[test]
-fn withdraw_fails_when_insufficient_liquidity() {
+fn redeem_fails_when_insufficient_liquidity() {
```

Comment:
should be in another unit tests, this is for testing normal redeem

Reply:
done